junit/src/main/java/org/kaazing/k3po/junit/rules/Latch.java

```diff
@@ -139,9 +139,10 @@ boolean hasException() {
 
     void notifyException(Exception exception) {
         this.exception = exception;
-        prepared.countDown();
-        startable.countDown();
-        finished.countDown();
+        // Commented because of issue https://github.com/k3po/k3po/issues/391
```

Comment:
I'm not following this, and based on the code I don't think you have gotten a fix yet for this, I would suggest pulling this out of the PR and putting in a different one